packages/icons/src/components/CryptoIcon.tsx

```diff
@@ -12,6 +12,7 @@ type CryptoIconProps = {
 const cryptoIconSizes = {
     extraSmall: 16,
     small: 24,
+    medium: 40,
```

Comment:
Seems redundant are those 2 px really noticeable?

Reply:
Agree, I would prefer use `large` for now

Reply:
[9b7e8d5](https://github.com/trezor/trezor-suite/pull/6199/commits/9b7e8d5c17992e6bab0c9737ee1651af0184bd69)